terraform/gcs/cloud_sql.tf

```
@@ -0,0 +1,73 @@
+# Create SQL instance
+resource "google_sql_database_instance" "postgres_instance" {
+  name             = "postgres-instance"
+  region           = "us-west1"
+  database_version = "POSTGRES_15"
+
+  settings {
+    tier = "db-f1-micro"
+  }
+
+  # todo: set `deletion_protection` depending on the environment
```

🐧
```suggestion
  # TODO: set `deletion_protection` depending on the environment
```
Makes it more prominent (and also consistent style)